addons/payment_payulatam/controllers/main.py

```diff
@@ -42,12 +42,13 @@ def payulatam_webhook(self, **data):
             'transactionState': data.get('state_pol'),
             'message': data.get('response_message_pol'),
             'lapTransactionState': lapTransactionState,
+            'payulatam_is_confirmation_page': True,
```

Comment:
Actually, you're modifying the `data`, which is a bit dirty.
Couldn't you add this variable to the context of `payment.transaction` just below?

Reply:
I hesitated to use this approach as indeed I thought the solution that I had a bit dirty too. I did test it experimentally and it looks to be equivalent. I did the update :)